libraries/AP_Camera/AP_Camera.h

```diff
@@ -69,7 +69,10 @@ class AP_Camera {
 
     // return true if we are using a feedback pin
     bool using_feedback_pin(void) const { return _feedback_pin > 0; }
-    
+
+    // location - get location of gimbal position considering offsets to CG
```

Comment:
The current_loc parameter in APMRover.cpp and ArduCopter.cpp..To be honest I do not know which is which. Can you clarify and I will correct variable names and comments?